planning/behavior_path_planner/autoware_behavior_path_static_obstacle_avoidance_module/include/autoware/behavior_path_static_obstacle_avoidance_module/scene.hpp

```diff
@@ -117,9 +118,44 @@ class StaticObstacleAvoidanceModule : public SceneModuleInterface
    */
   void updateRegisteredRTCStatus(const PathWithLaneId & path)
   {
+    // NOTE(odashima): Prevent duplication of control points of PlanningFactor.
+    auto filterCloseShiftLines =
```

Comment:
@Kotakku Please follow the ROS 2 coding conventions and use `snake_case` for all functions implemented from now on.
```suggestion
    const auto filter_close_shift_lines =
```